tests/nightly/apache_rat_license_check/README.md

```diff
@@ -0,0 +1,36 @@
+# Apache RAT License Check
+
+This is a nightly test that runs the Apache Tool RAT to check the License Headers on all source files
+ 
+### The .rat-excludes file
+This file lists all the files, directories and file formats that are excluded from license checks for various reasons.
+If you think something is wrong, feel free to change!
+
+### Nightly test script for license check
+Coming soon...
+
+### How to run the RAT check locally
+The following commands can be used to run a Apache RAT check locally - 
```

Comment:
Would it be possible to incorporate this with our ci-docker setup in order to allow people to execute a one-liner without requiring installation of any local dependencies?

Reply:
Yes, a script is coming with the nightly tests

Reply:
Great :)